### PR TITLE
aten::min aten::max

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: src/frontends/pytorch/src/op/min_max.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+std::shared_ptr<Node> get_all_axes(NodeContext& context) {
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/min_max.cpp]: please add this function to nameless namespace

Comment (eaidova) [reply src/frontends/pytorch/src/op/min_max.cpp]: moved to utils and reused in aten::sum

File: src/frontends/pytorch/src/op/tuple_construct.cpp [hunk 2]
```diff
@@ -12,11 +12,11 @@ namespace pytorch {
...
-    auto n_inputs = context.get_input_size();
-    FRONT_END_OP_CONVERSION_CHECK(
-        n_inputs == 1,
-        "prim::TupleConstruct conversion doesn't support cases when the number of inputs is not one.");
-    return {context.get_input(0)};
+    // auto n_inputs = context.get_input_size();
+    // FRONT_END_OP_CONVERSION_CHECK(
+    //     n_inputs == 1,
+    //     "prim::TupleConstruct conversion doesn't support cases when the number of inputs is not one.");
+    return {context.inputs()};
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/tuple_construct.cpp]: I don't think this is correct approach. You need to do this with transformation. Currently you return multiple outputs from operation that had one output, that is not correct.

Comment (eaidova) [reply src/frontends/pytorch/src/op/tuple_construct.cpp]: return back old behaviour, added transformation

File: src/frontends/pytorch/src/transforms/prim_tuple_construct_replacer.hpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#pragma once
```
Comment (mvafin) [inline src/frontends/pytorch/src/transforms/prim_tuple_construct_replacer.hpp:5]: copyright is missing

Comment (eaidova) [reply src/frontends/pytorch/src/transforms/prim_tuple_construct_replacer.hpp:5]: updated, thanks

**Review by mvafin — APPROVED** [review]

**Pull request merged**